SparkPost/2web2ui
# AC-736: Cancel accounts

### What Changed
 - Button to cancel plan on billing 
 - Button to renew plan if plan if pending cancellation
 - Pulled out brightback config

### How To Test
 - Navigate to `/account/billing`
 - Click on `Cancel Account`
 - Go through Brightback
 - Routed back and plan should be cancelled
 - Option should exist to renew plan
 - Should re-enable account
 - Downgrading to free should still work with brightback

### Notes
 - Currently staging and uat links will show downgrade link (which is the sandbox link).
   - Production should show the main cancellation link

### To Do
- [ ] Fix issue with new account info not being pulled after renewing account
